schemas/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment/jsonschema/1-0-0

```diff
@@ -64,25 +64,13 @@
                         }
                     },
                     "required": ["database", "uri"]
-                },
-                "httpUsername": {
-                    "description": "Optional username for basic HTTP authentication",
-                    "type": "string"
-                },
-                "httpPassword": {
-                    "description": "Optional password for basic HTTP authentication",
-                    "type": "string"
```

Comment:
why are we removing this completely?

Reply:
I spoke to @rzats about it. As I understand it:

* He never implemented the functionality
* He didn't document it

Therefore it makes more sense to remove it entirely given the release urgency.

If that's wrong or you disagree - @rzats or @BenFradet , please shout